core-compose/src/androidTest/java/com/squareup/workflow/ui/compose/ComposeViewFactoryTest.kt

```diff
@@ -47,18 +47,21 @@ class ComposeViewFactoryTest {
         }
 
     composeRule.setContent {
-      // This is valid Compose code, but the IDE doesn't know that yet so it will show an
-      // unsuppressable error.
-      RootView(viewEnvironment = viewEnvironment)
+      emitView(
+          ctor = ::RootView,
+          update = { it.setViewEnvironment(viewEnvironment) }
+      )
```

Comment:
Nit: More idiomatic to write like this:
```suggestion
      emitView(::RootView) {
        it.setViewEnvironment(viewEnvironment)
      }
```